JGActionSheet/JGActionSheet.m

```diff
@@ -506,6 +506,10 @@ @interface JGActionSheet () <UIGestureRecognizerDelegate> {
     BOOL _anchoredAtPoint;
     CGPoint _anchorPoint;
     JGActionSheetArrowDirection _anchoredArrowDirection;
+    UIView *_anchorPointView;
+    UICollectionView *_anchorViewParentCollectionView;
```

Comment:
Could you make this a readonly property, which gets the parent view and asserts it’s the correct class? (I’m just assuming less or more when it comes to maintaining state.)